docs/tutorials/basic_commands.md

```diff
@@ -151,7 +169,7 @@ needs to be renewed.
 ### Refresh
 
 In the last section, we mentioned that your contract can expire. Although free tokens never expire, if
-you buy an Ubuntu Pro subscription, and later need to renew the contract, how you can make your machine aware of
+you buy an Ubuntu Pro subscription and later need to renew the contract, how can you make your machine aware of
 it? You can do this through the `refresh` command:
 
 ```console
```

Comment:
Can't add a comment where I wanted to - in line 181, you say "After you modify your subscription for that". I got a bit confused about what this meant, and didn't realise you were referring to the website. Might be worth clarifying this (or linking to the same site, with "modify your subscription" as the link text).

Reply:
Done, let me know what you think

Reply:
LGTM :)